apps/web/components/ui/form/PhoneInput.tsx

```diff
@@ -15,6 +15,7 @@ type PhoneInputProps = {
 export const PhoneInput = ({ control, name, ...rest }: Props<PhoneInputProps>) => (
   <BasePhoneInput
     {...rest}
+    international
```

Comment:
When someone changes their country it still wipes their input but it now starts with the country code